resilience4j-spring-boot2/src/main/java/io/github/resilience4j/circuitbreaker/monitoring/endpoint/CircuitBreakerHystrixServerSideEvent.java

```diff
@@ -0,0 +1,128 @@
+        return eventStream.map(
+            cbEvent -> ServerSentEvent.<String>builder()
+                .id(cbEvent.getCircuitBreakerName())
+                .event(cbEvent.getEventType().name())
+                .data(data.apply(cbEvent, getCircuitBreaker(cbEvent.getCircuitBreakerName())))
+                .build()
+        );
+    }
+
+    @ReadOperation(produces = "text/event-stream")
+    public Flux<ServerSentEvent<String>> getHystrixStreamEventsFilteredByCircuitBreakerName(
+        @Selector String name) {
+
+        CircuitBreaker circuitBreaker = getCircuitBreaker(name);
+        Flux<CircuitBreakerEvent> eventStream = toFlux(circuitBreaker.getEventPublisher());
+        BiFunction<CircuitBreakerEvent, CircuitBreaker, String> data = getCircuitBreakerEventStringFunction();
+        return eventStream.map(
+            cbEvent -> ServerSentEvent.<String>builder()
+                .id(cbEvent.getCircuitBreakerName())
+                .event(cbEvent.getEventType().name())
+                .data(data.apply(cbEvent, circuitBreaker))
+                .build()
+        );
+    }
+
+    @ReadOperation(produces = "text/event-stream")
+    public Flux<ServerSentEvent<String>> getHystrixStreamEventsFilteredByCircuitBreakerNameAndEventType(
+        @Selector String name, @Selector String eventType) {
+
+        CircuitBreaker circuitBreaker = getCircuitBreaker(name);
+        Flux<CircuitBreakerEvent> eventStream = toFlux(circuitBreaker.getEventPublisher())
+            .filter(
+                event -> event.getEventType() == CircuitBreakerEvent.Type.valueOf(eventType.toUpperCase())
+            );
+        BiFunction<CircuitBreakerEvent, CircuitBreaker, String> data = getCircuitBreakerEventStringFunction();
+        return eventStream.map(cbEvent -> ServerSentEvent.<String>builder()
+            .id(cbEvent.getCircuitBreakerName())
+            .event(cbEvent.getEventType().name())
+            .data(data.apply(cbEvent, circuitBreaker))
+            .build()
+        );
+    }
+
+    private BiFunction<CircuitBreakerEvent, CircuitBreaker, String> getCircuitBreakerEventStringFunction() {
+        ObjectMapper jsonMapper = new ObjectMapper();
+        return (cbEvent, cb) -> {
+            try {
+                return jsonMapper.writeValueAsString(
+                    new CircuitBreakerHystrixStreamEventsDTO(cbEvent,
+                        cb.getState(),
+                        cb.getMetrics(),
+                        cb.getCircuitBreakerConfig()
+                    )
+                );
+            } catch (JsonProcessingException e) {
+                /* ignore silently */
+            }
+            return "";
+        };
+    }
+
+    private CircuitBreaker getCircuitBreaker(String circuitBreakerName) {
+        CircuitBreaker circuitBreaker = circuitBreakerRegistry.circuitBreaker(circuitBreakerName);
+        if (circuitBreaker == null && !circuitBreaker.getName().equalsIgnoreCase(circuitBreakerName)) {
```

Comment:
I think this doesn't need to be checked. 
`CircuitBreakerRegistry` create or get when you request a circuitbreaker. 
Therefore, it cannot be null, and it must be the same as the requested name.